code-challenges.rb

```diff
@@ -10,6 +10,17 @@
 num3 = 221
 # Expected output: '221 is odd'
 
+def odd_or_even num
+    if num % 2 == 0
+        puts "#{num} is even"
+    else
+        puts "#{num} is odd"
+    end
+end
+
+odd_or_even num1
+odd_or_even num2
+odd_or_even num3
```

Comment:
Logic looks great. It is a good practice to have the `puts` on the method invocation rather than the inner workings of the method.